### PR TITLE
Pull Request - Div Table

### PR DESCRIPTION
**Pull request by kubbafox**
A table with following features:
-Drag & Drop any rows or columns
-Sort
-Search
-Name tag popup
-Mobile Version

I have test the UI & layout through `http://localhost:4000/` and it works fine.

Known issue:
I have not started my SASS chapter yet, so the SASS code is not following the best practice.

### CODE REVIEW

File: assets/scripts/evolution/div_table.js [hunk 1]
```diff
@@ -0,0 +1,1158 @@
...
+      "name": "Anthony Mills",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "21%",
+      "contract_end_day": "9/7/2016",
+      "total_deals": 2,
+      "payment_cycle": "annually",
+      "outstanding_balance": "$10858.51",
+      "paid_amount": "$9096.66"
+    },
+    // {
```
Comment (jmatembu) [inline assets/scripts/evolution/div_table.js]: Hey @kubbafox,

Please clean up the code you have commented out.

Comment (kubbafox) [reply assets/scripts/evolution/div_table.js]: @jmatembu Sure. Sorry I missed it.

Comment (kubbafox) [reply assets/scripts/evolution/div_table.js]: @jmatembu Hi I have cleaned the code base. Please review the .js file again and let me know if you have further concerns. :)

**Comment by kubbafox** [comment]
Hi @jmatembu, do I have other outstanding issues in my pull request?

**Comment by jmatembu** [comment]
No @kubbafox 

Just merged. I love those tables, I see myself using them a lot. Thanks for the awesome work.

**Pull request merged**